get-user-locale.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module "get-user-locale" {
+  export function getUserLocale(): string;
+  export function getUserLocales(): string[];
+  export default getUserLocale;
```

Comment:
Here we don't need something like this:
```suggestion
  export default getUserLocale(): string;
```
? Idk, just makes sense to me, but I'm a total TS noob.

Reply:
It has been defined in line 2, so I just export it at default. We don't need define it again when define that export.